tracer/src/Datadog.Trace/ClrProfiler/AutoInstrumentation/AWS/SDK/RuntimePipelineInvokeAsyncIntegration.cs

```diff
@@ -50,6 +52,20 @@ public class RuntimePipelineInvokeAsyncIntegration
                 tags.Region = executionContext.RequestContext.ClientConfig.RegionEndpoint?.SystemName;
             }
 
+            if (scope.Span.Context.TraceContext.SamplingPriority == null)
+            {
+                scope.Span.Context.TraceContext.SetSamplingPriority(Tracer.Instance.TracerManager.Sampler?.GetSamplingPriority(scope.Span));
+            }
```

Comment:
I couldn't find the sampling priority set anywhere other than by using `Tracer.Instance.TracerManager.Sampler?.GetSamplingPriority(scope.Span)`. I expected it to be in `scope.Span.Context.TraceContext.SamplingPriority`, please let me know if I shouldn't be setting the sampling priority here, and just directly getting it from the sample (though I'd think the current span context should have the sampling priority?)